### PR TITLE
Second commit

### PR DESCRIPTION
**Pull request by Arpit-Tyagi**
Made some change according to the frontend of User Login and Register.

### CODE REVIEW

File: src/exception.filter.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import {
+  ExceptionFilter,
```
Comment (abhirocks550) [inline src/exception.filter.ts:2]: is this file imported anywherer?

File: src/main.ts [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
 
+
 async function bootstrap() {
-  const app = await NestFactory.create(AppModule);
+  const app = await NestFactory.create(AppModule, { cors: true });
+
   await app.listen(3030);
```
Comment (abhirocks550) [inline src/main.ts:8]: Fetch the port number as process.env.PORT_NUMBER || PORT_NUMBER from constant file.

File: src/users/users.controller.ts [hunk 3]
```diff
@@ -1,15 +1,24 @@
...
 @Controller('users')
 export class UsersController {
   constructor(private readonly usersService: UsersService) {}
 
-  @Post()
-  create(@Body() createUserDto: CreateUserDto) {
-    return this.usersService.create(createUserDto);
+  @Post('/register')
+  create(@Body() createUserDto: User) {
+
```
Comment (abhirocks550) [inline src/users/users.controller.ts:14]: keep the whitespaces and newlines only wherever required and keep it consistent.

File: src/users/users.service.ts [hunk 4]
```diff
@@ -10,9 +10,20 @@ export class UsersService {
 
   constructor( @InjectModel(User.name) private userModel: Model<UserDocument> ) {}
 
-  async create(createUserDto: CreateUserDto): Promise<User> {
+  async create(createUserDto: CreateUserDto): Promise<User> {  
 
-    return new this.userModel(createUserDto).save();
+            return  new this.userModel(createUserDto).save();
```
Comment (abhirocks550) [inline src/users/users.service.ts:15]: Use try-catch block so that runtime exceptions can be handled.

File: src/exception.filter.ts [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  Catch,
+  ArgumentsHost,
+  HttpException,
+  HttpStatus,
+} from '@nestjs/common';
+import { HttpAdapterHost } from '@nestjs/core';
+
+@Catch()
+export class AllExceptionsFilter implements ExceptionFilter {
```
Comment (abhirocks550) [inline src/exception.filter.ts:11]: Where does  Binding of Exception is happening?

File: src/users/users.service.ts [hunk 6]
```diff
@@ -10,9 +10,20 @@ export class UsersService {
...
+  
+}
+
+  async userLogin(createUserDto: User): Promise<User>  {
+    console.log(createUserDto);
```
Comment (abhirocks550) [inline src/users/users.service.ts:20]: Once testing in done with console. remove the console or if it required at staging add a comment

**Review by abhirocks550 — CHANGES_REQUESTED** [review]
Previous PR already merged so not able to review other files. Do not merge PR before reviewing.

**Comment by Arpit-Tyagi** [comment]
Made the required changes.
Can I merge the request??

**Pull request merged**